gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,58 @@
-"""Add docstring."""
+"""
+This class is used for storing the main functions requested from the Github.
 
+Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output,
+and colorization of text.
+"""
+import colorama as color
 
-# This class is used for storing the main functions requested
-# from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output,
-# description output, and colorization of text.
+color.init()
+
+
+def sort_checks_by_result(results):
+    """
+    Process results and determine if the check passed or failed.
+
+    Argument `results` is: list[(check_result_element1, check_result_element2),(...)]]
+    """
+    passed_checks = []
+    failed_checks = []
+    # iterate through results tuples
+    for i in results:

```

Comment:
```suggestion
    for result in results:

```

You may want to look at the most recent changes in the `out-feature-proper-checks-displayed` branch. It was mentioned that I should use descriptive variable names in my for loops so I implemented that. You will want to include those changes here!

Reply:
@lukel43 @pkelly55 This comment worries me a bit -- are you guys sharing your code without using Git? That may cause merge conflicts when one of your PRs is merged. We should be on the lookout, and unless your branches are based of each other (meaning that one of you branched off of the other), we'll probably need to make some fixes once the first PR is merged in.

Reply:
All the code has been being shared through git and by merging branches to stay up to date and be in the loop for changing code.